src/devices/pushok.ts

```diff
@@ -0,0 +1,269 @@
+import fz from '../converters/fromZigbee';
+import tz from '../converters/toZigbee';
+import * as exposes from '../lib/exposes';
+import {Definition, Fz, KeyValue, KeyValueAny, Publish, Tz, Zh} from '../lib/types';
+const e = exposes.presets;
+const ea = exposes.access;
+import {zigbeeOTA} from '../lib/ota';
+
+const valve_statuses = ['OFF', 'ON', 'MOVING', 'STUCK'];
+const battery_types = ['LIION', 'ALKALINE', 'NIMH'];
+const operation_modes = ['Monitor', 'Heater', 'Cooler'];
+
+const createConverter = (
```

Comment:
Could you refactor all of these to `modernExtend`? This replaces the to/fromZigbee and exposes. Examples: https://github.com/Koenkk/zigbee-herdsman-converters/blob/94467162420380dc8f5b8ca6d8e9ccb3fc4353c7/src/lib/lumi.ts#L1536